modules/navier_stokes/doc/content/source/problems/NavierStokesProblem.md

```diff
@@ -0,0 +1,97 @@
+# NavierStokesProblem
+
+The `NavierStokesProblem` solves the saddle point incompressible Navier-Stokes
+equations using Schur complement field split preconditioning. It focuses on the
+Least Squares Commutator (LSC) preconditioner, also known as the BFBt
+preconditioner, developed by Elman in [!citep](elman1999preconditioning) and
+[!citep](elman2006block). The PETSc manual page on the LSC preconditioner can be
+found [here](https://petsc.org/release/manualpages/PC/PCLSC/).
+
+A linearized version of the incompressible Navier-Stokes equations discretized
+with stable finite element pairings may be written as:
+
+\begin{equation}
+\begin{bmatrix}
+A & B^T\\
+B & 0
+\end{bmatrix}
+\begin{bmatrix}
+u\\
+p
+\end{bmatrix}
+=
+\begin{bmatrix}
+f\\
+0
+\end{bmatrix}
+\end{equation}
+
+where $u$ denotes velocity degrees of freedom, $p$ denotes pressure degrees of
+freedom, and $f$ denotes the effect of body forces not dependent on the
+velocity/pressure degrees of freedom (e.g. gravity). The LSC preconditioner
+proposed by Elman approximates the Schur complement inverse via
+
+\begin{equation}
+\hat{S}^{-1} := (B\hat{M}_u^{-1}B^T)^{-1}B\hat{M}_u^{-1}A\hat{M}_u^{-1}B_T(B\hat{M}_u^{-1}B_T)^{-1}
```

Comment:
if you could write M in the context of the A, B, B matrix above, I think it would make things clearer

it s some sort of preconditioning applied on B here imo

Reply:
The next line explains what M is:

> where $\hat{M}_u$ is a diagonal approximation of the velocity mass matrix

It doesn't have anything to do with A or B

> it s some sort of preconditioning applied on B here imo

What is this sentence referring to?

Reply:
It s referring to adding M in the expression

In the petsc manual for LSC, there is no M in the definition of S-1

Reply:
That's a PETSc issue. I didn't update that documentation in https://gitlab.com/petsc/petsc/-/merge_requests/6642